### DISCUSSION TITLE
Zignito TS011F OTA Failing

### DISCUSSION THREAD

**Discussion by jamesonuk** [Koenkk/zigbee2mqtt#15427] [Devices]
I have a device that has been picked up as https://www.zigbee2mqtt.io/devices/TS011F_plug_1.html and is affected by the random turning off issue.   When I try an OTA update I a get
`2022-12-07 15:38:53Failed to check if update available for 'Zignito Smart Plug' (No image available for imageType '5634')`

The manufacturer is coming up as `_TZ3000_1h2x4akh` and was sold as Ajax Online Zignito Zigbee Plug.

This is obviously the same TuYa device and is picked up fine but how can I get the OTA to work for it?

**Comment by Alex-Busch** [comment]
Did you found any solution?
